docs/plugins/cloud-gce.asciidoc

```diff
@@ -1,54 +1,53 @@
-Google Compute Engine Cloud Plugin for Elasticsearch
-====================================================
+[[cloud-gce]]
+=== GCE Cloud Plugin
 
-The GCE Cloud plugin allows to use GCE API for the unicast discovery mechanism.
+The Google Compute Engine Cloud plugin uses the GCE API for unicast discovery.
 
-In order to install the plugin, run: 
+[[cloud-gce-install]]
+[float]
+==== Installation
 
-```sh
-bin/plugin install elasticsearch/elasticsearch-cloud-gce/2.5.0
-```
+This plugin can be installed using the plugin manager:
 
-You need to install a version matching your Elasticsearch version:
+[source,sh]
+----------------------------------------------------------------
+bin/plugin install elasticsearch-cloud-gce
```

Comment:
Can we make a separate issue for that? Its absolutely terrible that 1.x continues to hold back 2.0. Until we change our attitude on this stuff, 2.0 will never make it out!